spyder/plugins/editor.py

```diff
@@ -913,6 +913,17 @@ def get_plugin_actions(self):
                 _("Unindent"), "Shift+Tab", icon=ima.icon('unindent'),
                 tip=_("Unindent current line or selection"),
                 triggered=self.unindent, context=Qt.WidgetShortcut)
+
+        self.text_uppercase_action = create_action(self,
+                _("Toggle Uppercase"), "Ctrl+Shift+U",
+                tip=_("Change to uppercase current line or selection"),
+                triggered=self.text_uppercase, context=Qt.WidgetShortcut)
+
+        self.text_lowercase_action = create_action(self,
+                _("Toggle Lowercase"), "Ctrl+U",
+                tip=_("Change to lowercase current line or selection"),
+                triggered=self.text_lowercase, context=Qt.WidgetShortcut)
+

```

Comment:
yeah, I just see an extra  definition here.

* https://github.com/spyder-ide/spyder/blob/master/spyder/config/main.py#L352-388

And here
* https://github.com/spyder-ide/spyder/blob/master/spyder/widgets/sourcecode/codeeditor.py#L457-477